src/seq/aminoacid.jl

```diff
@@ -81,7 +81,30 @@ function show(io::IO, aa::AminoAcid)
     end
 end
 
+# lookup table of 20 standard amino acids
+const threeletter_to_aa = @compat Dict(
+    "ALA" => AA_A, "ARG" => AA_R, "ASN" => AA_N, "ASP" => AA_D, "CYS" => AA_C,
+    "GLN" => AA_Q, "GLU" => AA_E, "GLY" => AA_G, "HIS" => AA_H, "ILE" => AA_I,
+    "LEU" => AA_L, "LYS" => AA_K, "MET" => AA_M, "PHE" => AA_F, "PRO" => AA_P,
+    "SER" => AA_S, "THR" => AA_T, "TRP" => AA_W, "TYR" => AA_Y, "VAL" => AA_V,
+)
+
+function Base.parse(::Type{AminoAcid}, s::String)
```

Comment:
Since you are trying to extend `parse` from Base, I believe you should import this function on `seq.jl` and only write `function parse(::Type{AminoAcid}, s::String)`. That's the way we do it when extending other functions from Base.